packages/docsearch-react/src/icons/SearchIcon.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 
-export function SearchIcon() {
+export function SearchIcon({ ...rest }) {
```

Comment:
not sure which way's best but if it's only for aria-hidden, maybe we can directly define on the svg?

Reply:
That works for me; looks like it's only used in these two places where it makes sense for both to have it hidden. Updated in latest.